home/tests.py

```diff
@@ -96,3 +99,100 @@ def question_tag_test_data(question_test_data):
     new_pair.tag = test_tag
     new_pair.save()
     return test_tag
+
```

Comment:
As I've suggested elsewhere, please group your tests in classes because you're going to have many of them in this file